spec/fixtures/github_api/inline_comments_pr_diff.diff

```diff
@@ -77,6 +77,17 @@ index 5817675..30499d7 100644
  * Show appropriate error message when GitHub repo was moved - KrauseFx
  * `danger plugins json [gem]` will now give gem metadata too - orta
  
+diff --git a/ios/Example/App Delegates/README.md b/ios/Example/App Delegates/README.md
+index 5817675..30499d7 100644
+--- a/ios/Example/App Delegates/README.md	
++++ b/ios/Example/App Delegates/README.md	
```

Comment:
Note: These two lines contain the trailing tabs. This is derived from a diff that I observed coming directly from Github Enterprise (although I reused the CHANGELOG.md contents here)